src/gridstack.ts

```diff
@@ -1072,13 +1072,12 @@ export class GridStack {
 
   /** @internal */
   private _initStyles(): GridStack {
-    if (this._stylesId) {
-      Utils.removeStylesheet(this._stylesId);
+    if (!this._stylesId) {

```

Comment:
I would move that create code to first line of _updateStyles() and change constructor to call it instead. Remove _initStyles()